app/views/shared/eligible_region_content_components/_professional_recognition.html.erb

```diff
@@ -5,12 +5,19 @@
   </p>
 <% end %>
 
-<% if region.status_check_online? && region.sanction_check_online? %>
-  <% if region.teaching_authority_other.present? %>
-    <%= raw GovukMarkdown.render(region.teaching_authority_other) %>
-  <% end %>
+<% if region.teaching_authority_other.present? %>
+  <%= raw GovukMarkdown.render(region.teaching_authority_other) %>
 <% end %>
 
+<% if region.status_check_online? || region.status_check_written?%>
+  <%= raw GovukMarkdown.render(region.teaching_authority_status_information) %>
+<% end %>
+
+<% if region.sanction_check_online? || region.sanction_check_written?%>
+  <%= raw GovukMarkdown.render(region.teaching_authority_sanction_information) %>
+<% end %>
```

Comment:
This content needs to appear in the right place according to the other content. For example, if the status check is online then we show the user:

> As your education department or authority has an online register of teachers, you’ll need to provide any registration numbers we need to check your record.

We'll need to show the `teaching_authority_status_information` directly below this because it's the status check is online.

Similarly, if the sanction check is online, we need to show `teaching_authority_sanction_information` at this point.

Further down, if the status or sanction check is written, we show the following:

> You’ll need to provide a certificate, which you can obtain by contacting:
> [address]

Below this section we need to show the user any sanction or status information, relevant only if the status or sanction is written (in a similar way to above for online).

Then only after all of this content has been written do we want to show the user `teaching_authority_other`, as this is designed to be a "catch all" field which is only used for content that doesn't fit in any of the other.

<hr />

For example, if the status check is online, and the status information is:

> The site can only be accessible within the US.

We need to ensure this appears next to the content that talks about an online register, and not next to any content referring to a written document and an address the teacher can use.